frontend/packages/shared/src/components/GiteaHeader/VersionControlButtons/components/ShareChangesPopover/CommitAndPushContent/FileChangesInfoModal/FileChangesInfoModal.test.tsx

```diff
@@ -93,11 +90,7 @@ const renderFileChangesInfoModal = (
     ...queriesMock,
     getRepoDiff,
   };
-  return render(
-    <ServicesContextProvider {...allQueries} client={createQueryClientMock()}>
-      <FileChangesInfoModal {...props} />
-    </ServicesContextProvider>,
-  );
+  return renderWithProviders(allQueries)(<FileChangesInfoModal {...props} />);
```

Comment:
Possible to use the same pattern we use in other tests, including remove the const above and importing `ServicesContextProps`
```suggestion
  return renderWithProviders({ ...queriesMock, getRepoDiff })(<FileChangesInfoModal {...props} />);
```